snake-kmp-benchmarks/src/jvmMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/benchmark/SnakeyamlEngineJvmLoadingTimeBenchmark.kt

```diff
@@ -0,0 +1,52 @@
+package it.krzeminski.snakeyaml.engine.kmp.benchmark
+
+import kotlinx.benchmark.*
+import okio.Path.Companion.toPath
+import okio.buffer
+import okio.use
+import org.snakeyaml.engine.v2.api.LoadSettings
+import org.snakeyaml.engine.v2.api.YamlUnicodeReader
+import org.snakeyaml.engine.v2.composer.Composer
+import org.snakeyaml.engine.v2.constructor.BaseConstructor
+import org.snakeyaml.engine.v2.constructor.StandardConstructor
+import org.snakeyaml.engine.v2.parser.ParserImpl
+import org.snakeyaml.engine.v2.scanner.StreamReader
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(BenchmarkTimeUnit.MILLISECONDS)
+class SnakeyamlEngineJvmLoadingTimeBenchmark {
+    @Param("")
+    var openAiYamlPath: String = ""
+
+    private val loadSettings = LoadSettings.builder().build()
+
+    private lateinit var constructor: BaseConstructor
+
+    @Setup
+    fun setUp() {
+        constructor = StandardConstructor(loadSettings)
+    }
+
+    @Benchmark
+    fun loadsOpenAiSchema(): Map<*, *> {
+        return with(fileSystem()) {
```

Comment:
question: why don't use just a standard java input stream instead of okio source which later converted to a stream? It is more fair in my opinion

Reply:
I wanted to keep the setup code as close to the KMP test as possible, to keep the overhead roughly the same to be able to compare both. But thanks for this comment, I'll give it more thought!